crates/crates_io_database/src/schema.rs

```diff
@@ -561,6 +561,28 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    /// Trusted Publisher OIDC configuration for GitHub Actions
+    github_oidc_configs (id) {
+        /// Unique identifier of the `github_oidc_configs` row
+        id -> Int4,
+        /// Unique identifier of the crate that this configuration is for
+        crate_id -> Int4,
+        /// GitHub user or organization that owns the repository
+        repository_owner -> Varchar,
+        /// Unique identifier of the user or organization that owns the repository
+        repository_owner_id -> Int4,
```

Comment:
Is there a reason we need to store both string and ID versions of this? Also, since this isn't a foreign key, I assume it is the GH ID and not any of our own internal IDs. If so, maybe we should tweak the comment to clarify that.

Reply:
> since this isn't a foreign key, I assume it is the GH ID and not any of our own internal IDs. If so, maybe we should tweak the comment to clarify that.

good catch. I actually updated the comments in the migration script already to reflect that, but forgot to regenerate the schema file 😅 



> Is there a reason we need to store both string and ID versions of this?

storing the string is useful to be able to show it in the list of active trusted publishing configs for a crate. but also, if you rename a repository then the ID will stay the same, but the name will be different. in that case we probably want the publish to fail and require a reconfiguration?